src/stories/prebuilt-ui.stories.tsx

```diff
@@ -75,8 +76,21 @@ const WherebyEmbed = ({
     video,
     virtualBackgroundUrl,
 }: Partial<WherebyEmbedAttributes>) => {
+    const elmRef = useRef<WherebyEmbedElement>(null);
+    const [cameraEnabled, setCameraEnabled] = useState(false);
```

Comment:
```diff
-const [cameraEnabled, setCameraEnabled] = useState(false);
+const [cameraEnabled, setCameraEnabled] = useState(video);
```

IMO this state's initial value should be the same as `WherebyEmbedElementExample.args.cameraAccess`, which is `true` initially.
Otherwise I see `camera: DISABLED` on the first render, even though it is enabled 🤔 

<img width="660" alt="Screenshot 2024-01-24 at 10 32 27" src="https://github.com/whereby/browser-sdk/assets/1159931/048b3e05-9243-4a13-a59d-922fbb670cf8">

Reply:
Good shout, I'll fix this 👍

Reply:
Done